test/cmdlineTests/standard_yul_single_file_via_urls/output.json

```diff
@@ -3,10 +3,10 @@
     [
```

Comment:
Why does this only have a single error now? Shouldn't it have the non-fatal followed by the fatal one?

Reply:
Fixed.

This was a leftover from my initial solution, where I made it not even run `compileYul()` if there were non-fatal errors. But then I realized that it's not good (they're non-fatal so we should continue). I forgot to update the test after that.